src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -75,10 +75,14 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
 
     private _getParamsScript(initializationData: PaypalCommerceInitializationData, cart: Cart): PaypalCommerceScriptParams {
         const { clientId, intent, isPayPalCreditAvailable, merchantId, attributionId } = initializationData;
+
         const disableFunding: DisableFundingType = [ 'card' ];
+        const enableFunding: EnableFundingType = [];
 
         if (!isPayPalCreditAvailable) {
-            disableFunding.push('credit');
+            disableFunding.push('credit', 'paylater');
```

Comment:
How come in one code path you need to disable credit + paylater, in another you need to enable?

Reply:
Because in first path merchant disabled credit for shopper and we need to pass credit and paylater method to 'disable-funding'. In another case merchant enabled credit for shopper and we need to pass credit and paylater method to 'enableFunding-funding' field. It's a requirements from paypal side.

Reply:
Yeah, I think would be better to rewrite this condition from !isPayPalCreditAvailable to isPayPalCreditAvailable

Reply:
`paylater` is the same as `credit` type, but visible for some countries only. We are expecting from PayPal that this will be always the same button but with different titles

Reply:
Thank you, spoke to @vneutrino privately. There's a need to explicitly enable and disable because the default value is different depending on which country it is used (EU vs US).